src/App.vue

```diff
@@ -217,4 +348,51 @@ nav a {
 nav a:first-of-type {
   border: 0;
 }
+
+#virtual-keyboard {
+  background-color: white;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  width: 100vw;
+  min-width: 500px; /* this stops the keys getting too close together for portrait mobile users */
+  height: calc(100vh - 50px);
+  z-index: 100;
```

Comment:
For a minor visual improvement add the rule `border-spacing: 0px;` to `#virtual-keyboard`